src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -138,27 +126,184 @@ public static void setDebug(boolean debug) {
     /**
      * The default duration in milliseconds spent not scanning between each bluetooth scan cycle when no ranging/monitoring clients are in the foreground
      */
-    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5*60*1000;
+    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5 * 60 * 1000;
 
     private long foregroundScanPeriod = DEFAULT_FOREGROUND_SCAN_PERIOD;
     private long foregroundBetweenScanPeriod = DEFAULT_FOREGROUND_BETWEEN_SCAN_PERIOD;
     private long backgroundScanPeriod = DEFAULT_BACKGROUND_SCAN_PERIOD;
     private long backgroundBetweenScanPeriod = DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD;
 
+    private static Logger sLogger = Loggers.empty();
```

Comment:
I would argue that in a release library there shouldn't be logging by default, and with the right amount of documentation I think people would know what to do for development. 

We could do `sLogger = BuildConfig.DEBUG ? Loggers.androidLogger() : Loggers.empty()` by default, but I think that would probably be more confusing. I'll update it to `androidLogger` implementation if you would like. I think I'll update for the loggers to lazy load as well, might as well have a smaller footprint if we can.

I did notice it seems like some of the docs in the website are out of date, I'll do a pass through and update what I can.